spring-ai-spring-boot-autoconfigure/src/test/java/org/springframework/ai/autoconfigure/minimax/FunctionCallbackWrapperIT.java

```diff
@@ -75,11 +76,12 @@ void functionCallTest() {
 
 	@Test
 	void streamFunctionCallTest() {
```

Comment:
same test failure as the other streaming tests

Reply:
> same test failure as the other streaming tests

hi, @markpollack i checked failed tests, sometimes, tool call might get split. i fixed this issue on PR https://github.com/spring-projects/spring-ai/pull/1276

Reply:
thank you!